src/test/test-exit-status.c

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include "exit-status.h"
+#include "tests.h"
+
+static void test_exit_status_to_string(void) {
+        log_info("/* %s */", __func__);
+
+        for (int i = -1; i <= 256; i++) {
+                const char *s, *class = NULL;
+
+                if (i >= 0 && (size_t) i < ELEMENTSOF(exit_statuses))
+                        class = exit_statuses[i].class & EXIT_STATUS_GLIBC ? "glibc" :
+                                exit_statuses[i].class & EXIT_STATUS_SYSTEMD ? "systemd" :
+                                exit_statuses[i].class & EXIT_STATUS_LSB ? "LSB" :
+                                exit_statuses[i].class & EXIT_STATUS_BSD ? "BSD" : NULL;
+
+                s = exit_status_to_string(i, EXIT_STATUS_FULL);
+                log_info("%d: %s%s%s%s",
+                         i, s ?: "-",
+                         class ? " (" : "", class ?: "", class ? ")" : "");
+        }
+}
```

Comment:
idea: drop the test again, make this a "systemd-analyze" verb instead, using the format-table code to output the nr, the name, and the origin